install/tutorial_6-x.md

```diff
@@ -22,7 +22,7 @@ Install prerequisites.  A clean Ubuntu system will need:
 
     sudo apt-get install ruby-dev build-essential libtinyxml-dev libboost-all-dev cmake mercurial pkg-config
 
-### ignition-math4:
+### gz-math4:
```

Comment:
Do we have a `gz-math4`? I think it's better to leave it as `ignition` for older tutorials